<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contactform</class>
 <widget class="QMainWindow" name="contactform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>781</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="NomLabel">
         <property name="text">
          <string>Nom : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="NomlineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="PrenomLabel">
         <property name="text">
          <string>Prénom : </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="PrenomlineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="MailLabel">
         <property name="text">
          <string>Mail : </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="MaillineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Tellabel">
         <property name="text">
          <string>Téléphone : </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="TellineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="EntrepriseLabel">
         <property name="text">
          <string>Entreprise : </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="EntrepriselineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="DateCrealabel">
         <property name="text">
          <string>Date de création : </string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="DateModiflaber">
         <property name="text">
          <string>Date de dernière modification :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="DateCrea">
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="DateModif">
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="enregisterPb">
         <property name="text">
          <string>Enregister</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="annulerPb">
         <property name="text">
          <string>Retour</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>751</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="photo">
       <property name="text">
        <string>l</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Photolabel">
       <property name="text">
        <string>Adresse de l'image : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PhotolineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
